src/tabs/discover/SearchFeed/__snapshots__/SearchFeed.tests.js.snap

```diff
@@ -561,16 +561,11 @@ Celebrate your freedom in Christ — in the car, shower, cubicle or wherever you
                             }
                           }
                         >
-                          <Image
-                            maxAspectRatio={1.78}
-                            minAspectRatio={1.2}
-                            onLoad={[Function]}
-                            source={Array []}
+                          <View
```

Comment:
Kind of concerned that this would change since we didn't make any changes to the `SearchFeed`.